cypress/e2e/iframe.cy.ts

```diff
@@ -0,0 +1,18 @@
+import {css} from "cypress/types/jquery";
+import {Iframe} from "../page/index";
+
+describe("uploading and download a file test", () => {
+  let iframe: Iframe;
+
+  before(() => {
+    iframe = new Iframe();
+  });
+
+  it("testing the title after upload file", () => {
+    const htmlTitle = "HTML Tutorial";
+    const cssTitle = "CSS Tutorial";
+    iframe.visit();
+    iframe.getFrameTitle(htmlTitle);
+    iframe.goToCssPageInFrame(cssTitle);
+  });
```

Comment:
we should have two separated tests, you can put the visit call in the before hook. The tests are:

1. To verify the HTM Tutorial title
2. To verify the CSS Tutorial title